crates/pathfinder/src/ethereum/log/fetch/forward.rs

```diff
@@ -153,7 +153,9 @@ where
                 }
             }
 
-            self.last_known = logs.last().cloned();
+            if let Some(last) = logs.last() {
+                self.last_known = Some(last.clone());
```

Comment:
Should here be some assertion/check that we are indeed going forward, or does that follow from some underlying property...?

Reply:
It follows that we are going forward because [to_block = from_block + stride](https://github.com/eqlabs/pathfinder/pull/97/files#R95). We are always starting at our last known position and going forwards.

Reply:
Hmm ok!